tmail-backend/mailbox/encrypted/api/src/test/scala/com/linagora/tmail/encrypted/AttachmentMetaDataSerializerTest.scala

```diff
@@ -0,0 +1,78 @@
+package com.linagora.tmail.encrypted
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import play.api.libs.json.{JsValue, Json}
+
+class AttachmentMetaDataSerializerTest {
+
+  @Test
+  def serializeShouldBeSuccess(): Unit = {
```

Comment:
You have a problem with those `Be` everywhere :) Remove it please ^^ (and in other tests names too)